src/routes/+layout.svelte

```diff
@@ -4,6 +4,19 @@
   import { loginStatus } from "$lib/stores/loginStatus";
   import { page } from "$app/stores";
   import "../app.css";
+  import { onMount } from "svelte";
+
+  function getCookieValue(name) {
+    const value = `; ${document.cookie}`;
+    const parts = value.split(`; ${name}=`);
+    if (parts.length === 2) return parts.pop().split(';').shift();
+  }
+
+
+  onMount(() => {
+    let myCookieValue = getCookieValue('access_token');
+    if (myCookieValue) { $loginStatus = true; }
```

Comment:
I guess we should validate the expiration time of token (by accessing exp field)
The token may be expired, but this code still will change the loginStatus to true.